src/useWindowSize.ts

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+
+interface WindowSize {
+  width: number;
+  height: number;
```

Comment:
```suggestion
  windowWidth: number;
  windowHeight: number;
```

[P2] 위와 같은 좀 더 명확한 네이밍은 어떨까요? 다른 분들의 의견도 부탁드려요!

cc. @jeongbaebang @Choozii

Reply:
내부 객체명은 useWindowSize에서 반환되는 객체명이므로, 좀 더 추상화된 ‘width’나 ‘height’ 같은 이름을 사용하는 것도 괜찮아 보입니다.

Reply:
내부 객체명은 useWindowSize에서 반환되는 객체명이므로, 좀 더 추상화된 ‘width’나 ‘height’ 같은 이름을 사용하는 것도 괜찮아 보입니다.

Reply:
@foresec width와 height로 가시죠!